app/assets/javascripts/govuk_publishing_components/lib/govspeak/magna-charta.js

```diff
@@ -0,0 +1,422 @@
+// non-jquery version of magna-charta: https://github.com/alphagov/magna-charta
+window.GOVUK = window.GOVUK || {}
+window.GOVUK.Modules = window.GOVUK.Modules || {};
+
+(function (Modules) {
+  function MagnaCharta () { }
+
+  MagnaCharta.prototype.start = function ($module, options) {
+    this.$module = $module[0]
+    this.options = {
+      outOf: 65,
+      applyOnInit: true,
+      toggleText: 'Toggle between chart and table',
+      autoOutdent: false,
+      outdentAll: false,
+      toggleAfter: false, // BOOL set TRUE to append the toggle link
+      returnReference: false // for testing purposes
+    }
+
+    for (var k in options) this.options[k] = options[k]
+    this.detectIEVersion()
+
+    // if it's IE7 or less, we just show the plain tables
```

Comment:
Leaving aside what browser GOV.UK supports, I think adding a reason helps make this more understandable.
```suggestion
    // Magna Charta doesn't work in IE7 or less - so plain tables are shown to those browsers.
```

Reply:
Thanks for the suggestion, have changed. Again, the perils of adding someone else's code...